source/utils/ConfigManager.h

```diff
@@ -130,6 +134,10 @@ class ConfigManager : public Ogre::Singleton<ConfigManager>
     std::map<const CreatureDefinition*, std::vector<const SpawnCondition*> > mCreatureSpawnConditions;
     std::map<const std::string, std::vector<const CreatureMood*> > mCreatureMoodModifiers;
     std::map<const std::string, std::vector<std::string> > mFactionSpawnPool;
+
+    //! \brief Stores the faction default worker creature definition.
+    std::map<const std::string, std::string> mFactionDefaultWorkerClass;
+
```

Comment:
Not that it's going to make a huge difference, but the strings in the map probably shouldn't be const, as that may prevent move-construction if the map has to reallocate memory.